users/tests/test_allauth.py

```diff
@@ -0,0 +1,426 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, unicode_literals
+
+from importlib import import_module
+from requests.exceptions import HTTPError
+
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.core import mail
+from django.test.client import RequestFactory
+from django.test.utils import override_settings
+from django.urls import reverse
+
+from allauth.account import app_settings as account_settings
+from allauth.account.adapter import get_adapter
+from allauth.account.models import EmailAddress, EmailConfirmation
+from allauth.account.signals import user_signed_up
+from allauth.socialaccount.models import SocialAccount, SocialToken
+from allauth.socialaccount.tests import OAuth2TestsMixin
+<<<<<<< HEAD
+from allauth.tests import MockedResponse, patch
+from allauth.socialaccount.providers.google.provider import GoogleProvider
+from pytest_django.asserts import assertTemplateUsed
+import pytest
+
+@pytest.mark.django_db
+# Using this annotation to ensure what settings defined for the tests of google sign-in features in this class
+=======
+from allauth.tests import MockedResponse, TestCase, patch
+
+from allauth.socialaccount.providers.google.provider import GoogleProvider
+
+# from allauth.socialaccount import views
+# from allauth.socialaccount.providers.google import views
+
+
+>>>>>>> 42f9e5e (Fixtures and tests for sign in with google feature)
+@override_settings(
+    SOCIALACCOUNT_AUTO_SIGNUP=True,
+    ACCOUNT_SIGNUP_FORM_CLASS=None,
+    ACCOUNT_EMAIL_VERIFICATION=account_settings.EmailVerificationMethod.MANDATORY,
+)
+<<<<<<< HEAD
+class GoogleTests(OAuth2TestsMixin):
```

Comment:
You did not rebase correctly

Reply:
Yes not sure how this file got uploaded I erased it some kind of mistake of course..